Sources/Benchmark/BenchmarkSetting.swift

```diff
@@ -14,23 +14,29 @@
 
 public enum BenchmarkSetting {
     case iterations(Int)
+    case maxIterations(Int)
     case warmupIterations(Int)
     case filter(String)
+    case minTime(Double)
```

Comment:
```suggestion
    case minTime(seconds: Double)
```

to help clarify what units the `Double` is in.